docs/devops/gcp/README.md

```diff
@@ -42,11 +42,16 @@ gcloud services enable run.googleapis.com
 gcloud services enable iam.googleapis.com
 gcloud services enable secretmanager.googleapis.com
 gcloud services enable artifactregistry.googleapis.com
+gcloud services enable cloudresourcemanager.googleapis.com
 ```
 
 ## Provision infrastruture
 
-From `infra/terraform/gcp` folder run these commands from your terminal:
+To allow Terraform to provision services on GCP we need to give it a bucket to store it's state.
```

Comment:
apologies for nitpicking on this stuff. it's -> its, right?

Reply:
nitpicking is always fine